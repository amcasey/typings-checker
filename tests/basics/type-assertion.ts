// This contains a type assertion which matches.

let v: number;

// $ExpectType number
v
