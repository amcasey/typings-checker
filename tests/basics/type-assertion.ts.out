tests/basics/type-assertion.ts: 1 / 1 checks passed.
